codespell_lib/_codespell.py

```diff
@@ -776,7 +785,13 @@ def _script_main():
 
 def main(*args):
     """Contains flow control"""
-    options, parser = parse_options(args)
+    options, parser, used_cfg_files = parse_options(args)
+
+    # Report used config files
+    if len(used_cfg_files) > 0:
```

Comment:
Too late for this PR, but you could have used `if used_cfg_files` instead of `if len(used_cfg_files) > 0`.

Rationale from Deepsource.io:
> Using the `len` function to check if a sequence is empty is not idiomatic and can be less performant than checking the truthiness of the object.
> 
> `len` doesn't know the context in which it is called, so if computing the length means traversing the entire sequence, it must; it doesn't know that the result is just being compared to 0. Computing the boolean value can stop after it sees the first element, regardless of how long the sequence actually is.